src/components/modals/CustomModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { Modal } from "antd";
+import { Imodal } from "../../types/types";
+import StepOneRegistration from "./StepOneRegistration";
+import StepTwoRegistration from "./StepTwoRegistration";
+
+const CustomModal: React.FC<Imodal> = ({
```

Comment:
There's a typo here `Imodal`

Reply:
issue solved in commit d23730ea5a20ee7c717bab26107db18ccebe94fd.